src/components/category.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import Loader from '../images/loader.svg'
+
+class Fetch extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      ctg: [],
+      isloading: true,
+    }
+  }
+
+  componentWillMount() {
+    const id = this.props.match.params.id
+    fetch(`http://127.0.0.1:3000/api/v1/category/${id}`)
+      .then(res => res.json())
+      .then(output => {
+        this.setState({ ctg: output, isloading: false })
+        console.log(output)
+      })
+  }
+
+  render() {
+    if (this.state.isloading) {
+      return (
+        <div>
```

Comment:
Make a separate `Loader` component, import it and then use it here and everywhere else.